frontend/public/components/utils/resource-log.jsx

```diff
@@ -436,3 +435,16 @@ ResourceLog.propTypes = {
   resource: PropTypes.object.isRequired,
   resourceStatus: PropTypes.string.isRequired,
 };
+
+LogControls.propTypes = {
+  isFullscreen: PropTypes.bool,
+  dropdown: PropTypes.node,
+  status: PropTypes.string,
+  resource: PropTypes.object,
+  containerName: PropTypes.string,
+  podLogLinks: PropTypes.array,
+  namespaceUID: PropTypes.string,
+  toggleStreaming: PropTypes.func,
+  onDownload: PropTypes.func,
+  toggleFullscreen: PropTypes.func,
```

Comment:
```suggestion
  podLogLinks: PropTypes.arrayOf(PropTypes.object), // K8sResourceKind
  namespaceUID: PropTypes.string,
  toggleStreaming: PropTypes.func,
  onDownload: PropTypes.func.isRequired,
  toggleFullscreen: PropTypes.func.isRequired,
```
I think `onDownload` and `toggleFullscreen` are explicitly used on buttons and if they are showing up, they should have a callback imo.

Also I think `arrayOf` & the comment is needed to just expose it (until this file is TSX with types).

Reply:
Done